2019/data-structures/presentations/5_week5/index.md

```diff
@@ -445,6 +446,7 @@ struct DoubleNode
 {
     double data;
     DoubleNode *next;
+    DoubleNode *prev;
```

Comment:
No `prev` pointer here. `DoubleNode` means the node holds a `double` data type. The node that has `prev` pointer is called **doubly**.

Reply:
Changes applied